Formula/g/gyb.rb

```diff
@@ -122,20 +154,14 @@ class Gyb < Formula
     sha256 "df7aa8afb0148fa78488e7899b2c59b5f4ffcfa82e6c54ccb9dd37c1d7b52d54"
   end
 
-  def python3
-    "python3.12"
-  end
-
   def install
     # change user config location from default of executable own path
     inreplace "gyb.py", "default=getProgPath()",
                         "default='#{pkgetc}'"
-    venv = virtualenv_create(buildpath, python3)
+    venv = virtualenv_create(buildpath, "python3.12")
     venv.pip_install resources
 
-    ENV.append_path "PYTHONPATH", buildpath/Language::Python.site_packages(python3)
-    pyinstaller = Formula["pyinstaller"].opt_bin/"pyinstaller"
-    system pyinstaller, "gyb.spec"
+    system "bin/pyinstaller", "gyb.spec"
```

Comment:
Wasn't sure if there's a better DSL helper method to say "the bin path of the virtualenv we're in" here.